docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -2794,11 +2794,12 @@ Content-Type: application/json
 ]
 ```
 
-
-
 Query Parameters:
 
-- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the networks list. Available filters: `name=[network-names]` , `id=[network-ids]`
+- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the networks list. Available filters:
```

Comment:
@WeiZhang555 I'm not sure it is necessary to mention the format if there are only two types of filters available. How about:

```markdown
- **filters** - JSON encoded network list filter. The value is one of: `name=[network-names]` or `id=[network-ids]`
```